{"frames":[{"func":"max_temp_raster_crop %>% as(., \"SpatialPolygonsDataFrame\")","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"as(., \"SpatialPolygonsDataFrame\")","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"asMethod(object)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"rasterToPolygons(from)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"cbind(xyFromCell(x, 1:ncell(x)), getValues(x))","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"getValues(x)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"getValues(x)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":".local(x, ...)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"readAll(x)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"readAll(x)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":".readRasterBrickValues(object, 1, object@nrows)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"readStart(object)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"readStart(object)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":".openConnection(x, ...)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"file(fn, \"rb\")","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0}],"message":"Error in file(fn, \"rb\") : cannot open the connection\n"}